yb-voyager/cmd/importData.go

```diff
@@ -1024,7 +1024,7 @@ func findCopyCommandForDebeziumExportedFiles(tableName, dataFilePath string) (st
 	defer df.Close()
 	columnNames := quoteColumnNamesIfRequired(df.GetHeader())
 	stmt := fmt.Sprintf(
-		`COPY %s(%s) FROM STDIN WITH (FORMAT CSV, DELIMITER ',', HEADER, ROWS_PER_TRANSACTION %%v);`,
+		`COPY %s(%s) FROM STDIN WITH (FORMAT TEXT, DELIMITER E'\t', ROWS_PER_TRANSACTION %%v);`,
```

Comment:
the default delimiter is tab, so I think we can do without explicitly specifying it

Reply:
for CSV as well the default is a comma and we were explicitly mentioning it.